bot/code_coverage_bot/uploader.py

```diff
@@ -45,10 +46,10 @@ def gcp(repository, revision, report, platform, suite):
     logger.info("Uploaded {} on {}".format(path, bucket))
 
     # Trigger ingestion on backend
-    retry(
-        lambda: gcp_ingest(repository, revision, platform, suite),
-        retries=10,
-        wait_between_retries=60,
+    @tenacity.retry(
+        retry = retry_if_exception(lambda: gcp_ingest(repository, revision, platform, suite)),
+        stop=stop_after_attempt(10),
+        wait=wait_fixed(60)
```

Comment:
Nit: linting here will probably complain, as you are using ` = ` once and `=` without spaces twice. Use pre-commit to format everything.